dist/acme/index.html

```diff
@@ -0,0 +1,54 @@
+<!doctype html>
```

Comment:
Oops, you added the dist-folder to the commit as well. Delete these files, they are generated and should not be commited.

There is a file called `.gitignore` - if you add a line saying `dist` to it, these folder will be ignored in the future.